### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Startpage](https://hosted.weblate.org/projects/jamulus/1-index/).

It also includes the following components:

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/Navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/Unregistered-Servers](https://hosted.weblate.org/projects/jamulus/unregistered-servers/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/Copyright notices](https://hosted.weblate.org/projects/jamulus/copyright-notices/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/General strings](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)

Current translation status:

Weblate translation status

### CODE REVIEW

**Review by jujudusud — APPROVED** [review]

File: _translator-files/po/es/1-index.po [hunk 1]
```diff
@@ -128,19 +107,19 @@ msgstr "¿Quieres involucrarte?"
...
 msgid "Ideas? Found a bug? Want to contribute some code or help [translating](https://github.com/jamulussoftware/jamulus/blob/main/docs/TRANSLATING.md) Jamulus into your language? Since Jamulus is [free and open source software](https://www.gnu.org/philosophy/free-sw.en.html) (FOSS) licensed under the [GPL](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html) you can help us!"
-msgstr ""
+msgstr "¿Ideas? Has encontrado un bug? ¿Quieres contribuir con código o ayudar a [traducir](https://github.com/jamulussoftware/jamulus/blob/main/docs/translating.md) Jamulus a tu idioma? Puesto que Jamulus es [software libre y de código abierto](https://www.gnu.org/philosophy/free-sw.en.html) (FOSS) bajo la licencia [GPL](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html) ¡nos puedes ayudar!"
 
 #. type: Content of: <div><div><p><div><br><br>
 #: ../wiki/en/misc/1-index.md:55
 #, no-wrap
 msgid "Take a look at our [contribution guidelines](wiki/Contribution) to find out how. Everybody is welcome!"
-msgstr ""
+msgstr "Echa un vistazo a nuestra [guía para contribuciones](wiki/Contribución) para saber cómo. ¡Cualquier persona es bienvenida!"
```
Comment (ann0see) [inline _translator-files/po/es/1-index.po]: Not sure if that's the correct link:

```suggestion
msgstr "Echa un vistazo a nuestra [guía para contribuciones](wiki/Contribution) para saber cómo. ¡Cualquier persona es bienvenida!"
```

**Review by ann0see — CHANGES_REQUESTED** [review]

File: _translator-files/po/es/1-index.po (hunk 1 above)
Comment (ignotus666) [reply _translator-files/po/es/1-index.po]: Thanks - corrected. There's a user on Weblate who has the exasperating habit of changing stuff that was already ok. He's well-intentioned but it's exhausting, as I have to go after him and comb through everything he touches... I've repeatedly told him to leave links alone, to not arbitrarily change things on a whim without any consistency... If it keeps going on I'll have to end up blocking him.

**Review by ignotus666 — APPROVED** [review]

File: _translator-files/po/de/1-index.po [hunk 2]
```diff
@@ -79,10 +64,9 @@ msgstr "Was ist Jamulus?"
 
 #. type: Content of: <div><div><div><div>
 #: ../wiki/en/misc/1-index.md:23
-#, fuzzy, no-wrap
-#| msgid "       Jamulus lets you play, rehearse, or jam with your friends, your band, or anyone you find online. Play together remotely in time with high quality, low-latency sound on a normal broadband connection. "
+#, no-wrap
 msgid "Jamulus lets you play, rehearse, or jam with your friends, your band, or anyone you find online. Play together remotely in time with high quality, low-latency sound on a normal broadband connection. [Download it here!](wiki/Getting-Started)\n"
-msgstr "       Mit Jamulus kannst du mit deinen Freunden, deiner Band oder jedem, den du online findest, spielen, proben oder jammen. Spiele aus der Ferne im Takt mit hochqualitativem Sound mit geringer Latenz über eine normale Breitbandverbindung. "
+msgstr "Jamulus ermöglicht es dir, mit deinen Freunden, deiner Band oder jedem, den du online findest, zu spielen, zu proben oder gemeinsam zu musizieren. Spiele zusammen in Echtzeit mit hochwertigem, geringem Latenz-Sound über eine normale Breitbandverbindung. [Lade es hier herunter!](wiki/Getting-Started)\n"
```
Comment (ann0see) [inline _translator-files/po/de/1-index.po]: This needs some thought.

File: _translator-files/po/de/1-index.po [hunk 3]
```diff
@@ -99,9 +83,8 @@ msgstr "Jamulus weltweit"
 #| "    "
 msgid "All over the world Jamulus allows choirs to rehearse and rock bands to play. Jamulus brings folk and classical musicians together. It's being used for remote music lessons, in schools and universities, in private and in public — all in real time on the Internet, as if you were there in person.\n"
 msgstr ""
-"    Überall auf der Welt können dank Jamulus Chöre zusammen proben und Rockbands ihre Gigs aufführen.\n"
+"Überall auf der Welt, können dank Jamulus Chöre zusammen proben und Rockbands ihre Gigs aufführen.\n"
```
Comment (ann0see) [inline _translator-files/po/de/1-index.po]: No. That's wrong -> revert

**Review by ann0see — CHANGES_REQUESTED** [review]
Unfortunately German needs updates.

File: _translator-files/po/nl/1-index.po [hunk 4]
```diff
@@ -116,6 +117,8 @@ msgid ""
 "Check out the [documentation](wiki/Getting-Started) and consider the [troubleshooting section](wiki/Client-Troubleshooting)!\n"
 "You can also ask on the [forums](https://github.com/jamulussoftware/jamulus/discussions).\n"
 msgstr ""
+"Lees de [documentatie](wiki/Getting-Started) en overweeg ook nog een bezoek aan de [troubleshooting sectie](wiki/Client-Troubleshooting)!\n"
```
Comment (ann0see) [inline _translator-files/po/nl/1-index.po:120]: @henkdegroot please check if that's ok.

File: _translator-files/po/it/1-index.po [hunk 5]
```diff
@@ -82,10 +83,9 @@ msgstr "Cos'è Jamulus?"
 
 #. type: Content of: <div><div><div><div>
 #: ../wiki/en/misc/1-index.md:23
-#, fuzzy, no-wrap
-#| msgid "       Jamulus lets you play, rehearse, or jam with your friends, your band, or anyone you find online. Play together remotely in time with high quality, low-latency sound on a normal broadband connection. "
+#, no-wrap
 msgid "Jamulus lets you play, rehearse, or jam with your friends, your band, or anyone you find online. Play together remotely in time with high quality, low-latency sound on a normal broadband connection. [Download it here!](wiki/Getting-Started)\n"
-msgstr "       Jamulus ti consente di suonare, provare o suonare con i tuoi amici, la tua band o chiunque trovi online. Suona insieme da remoto a tempo con audio di alta qualità e bassa latenza su una normale connessione a banda larga. "
+msgstr "Jamulus ti consente di suonare, provare o suonare con i tuoi amici, la tua band o chiunque trovi online. Suona insieme da remoto a tempo con audio di alta qualità e bassa latenza su una normale connessione a banda larga. [Scaricalo qui!](wiki/Getting-started)\n"
```
Comment (ann0see) [inline _translator-files/po/it/1-index.po]: Fixed a link. Should still be checked.

**Review by ann0see — APPROVED** [review]
Seems to be ok, but I'd still like to have an ok from a few others.

**Comment by trebmuh** [comment]
ok

File: _translator-files/po/nl/1-index.po (hunk 4 above)
Comment (henkdegroot) [reply _translator-files/po/nl/1-index.po:120]: Looks okay to me.

Comment (ann0see) [reply _translator-files/po/nl/1-index.po:120]: Thanks for the confirmation

**Pull request merged**